tarynmartin/shopify-challenge
# Refactoring

#### What’s this PR do?
- Remove setLocalStorage function and put functionality within useEffect for banner
- Add messages for Results and Nominations for when those arrays are empty

#### Where should the reviewer start?
- App.js to see updated way to set local storage
- Results & Nominations.js and associated css files for messages to display and associated styling

#### How should this be manually tested?
- in terminal, run npm start
- in browser, see messages on load
- Perform a search to see results message disappear
- Nominate a movie to see nominations message disappear
